jerry-core/debugger/debugger.c

```diff
@@ -981,4 +994,24 @@ jerry_debugger_send_exception_string (ecma_value_t exception_value) /**< error v
   return result;
 } /* jerry_debugger_send_exception_string */
 
+/**
+ * Send a signal about the engine waiting status to the client.
+ */
+void
+jerry_debugger_send_wait_status (bool is_waiting)
```

Comment:
Perhaps the argument is not needed anymore. In that case we can simply use jerry_debugger_send_type.